meteor-app/imports/startup/server/autopay.test.js

```diff
@@ -101,12 +101,14 @@ if (Meteor.isServer) {
     it('sends the autopay notice', () => {
       expect(cart.autoPayNoticeDate).to.be.equal(undefined)
       expect(() => {
-        Meteor.call('autoPayNotice')
+        // TODO - This is problematic, because it's trying send an email, but sendgrid won't do it
```

Comment:
Hey Mike, is this something we need to do now or is it something we can do later?